docker/install-dependencies.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/bash
+
+set -euo pipefail
+
+# The URL to RCM Tools Fedora repos
+rcm_tools_repos=$1
```

Comment:
I think you'll need to set a default value here. Empty string should do it.

Reply:
If build argument `rcm_tools_repo_file` is not set to `docker build`, `$1` will be expanded to an empty string eventually. This is an expected behavior. Sorry, I'm a little confused. I'm afraid it is unnecessary to set a default value here.

Reply:
The `-u` flag will cause an error when accessing an undefined variable, in this case `$1`:
```
🐚 cat test.sh 
#!/bin/bash
set -euo pipefail
rcm_tools_repos=$1
echo 'It works!'
🐚 ./test.sh http://example.com/spam.repo
It works!
🐚 ./test.sh 
./test.sh: line 4: $1: unbound variable
```

You want this line to be `rcm_tools_repos=${1:-}`

Reply:
I get it. Thanks. :)